### PR TITLE
NO-JIRA:  Allow the command in wait_for_cmd_amount to receive arguments to pass to the inner function

### PR DESCRIPTION
**Pull request by ori-amizur**
This was already assumed but it is not the case now

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md
/cc @paul-maidment

### CODE REVIEW

File: deploy/operator/utils.sh [hunk 1]
```diff
@@ -123,10 +123,10 @@ function wait_for_cmd_amount() {
     interval="$2"
     cmd="$3"
 
-    until [ $(${cmd} | wc -l) -eq ${amount} ]; do
+    until [ $(${cmd} "${@:4}" | wc -l) -eq ${amount} ]; do
```
Comment (paul-maidment) [inline deploy/operator/utils.sh:126]: Would prefer to create a var in a similar fashion to 'cmd'

I.e 

arguments = "${@:4}"

Comment (ori-amizur) [reply deploy/operator/utils.sh:126]: Since it is multiple arguments, it seems better this way

Comment (paul-maidment) [reply deploy/operator/utils.sh:126]: Ok

**Comment by adriengentil** [comment]
/hold
to prevent the merge before all jobs have been tested

**Pull request merged**